src/tests/agent_trajectory/feed_explore.py

```diff
@@ -0,0 +1,116 @@
+import asyncio
+import unittest
+
+from langchain_core.messages import HumanMessage
+from openagent.agents.feed_explore import feed_explorer_agent
+from openagent.conf.llm_provider import set_current_llm
+
+class TestFeedExploreAgent(unittest.TestCase):
+    def setUp(self):
+        set_current_llm("gpt-3.5-turbo")
+
+    def test_query_defi_activities(self):
+        """
+        Test querying recent DeFi activities for a specific Ethereum address.
+        Validates that the response contains the correct address and is handled by the 'DeFiExecutor'.
+        """
+        async def async_test():
+            events = feed_explorer_agent.astream_events({"messages": [
+                HumanMessage(content="Show me recent DeFi activities for address 0x742d35Cc6634C0532925a3b844Bc454e4438f44e", name="human")]},
+                version="v1")
+
+            async for event in events:
+                if event['event'] == 'on_tool_end':
+                    event_data_input_ = event['data']['input']
+                    self.assertEqual(event['name'], 'DeFiExecutor')
+                    self.assertIn('address', event_data_input_)
```

Comment:
The activity_type is incorrect and should be verified.

<img width="1613" alt="image" src="https://github.com/user-attachments/assets/060aec7e-23ea-4a58-988a-3c35c21cc18c">

Reply:
I added an assert check statement here to check the `activity_type`.

Also, I added the logic for the case of `activity_type = all` (while no specific DeFi activity is defined and the DeFiExecutor is triggered, then it searches for all DeFi activities.